store/cassandra/store_cassandra.go

```diff
@@ -80,12 +81,15 @@ var (
 	errmetrics = cassandra.NewErrMetrics("store.cassandra")
 )
 
-/*
-https://godoc.org/github.com/gocql/gocql#Session
-Session is the interface used by users to interact with the database.
-It's safe for concurrent use by multiple goroutines and a typical usage scenario is to have one global session
-object to interact with the whole Cassandra cluster.
-*/
+type ChunkReadRequest struct {
```

Comment:
why is the definition of `ChunkReadRequest` moved here, but `ChunkWriteRequest` remains where it was?

Reply:
`chunkWriteRequest`s are used in the `store` interfaces `Add` method so all backend stores use them. The `chunkReadRequest` is a Cassandra specific implimentation detail